src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -77,7 +77,7 @@
   </AbstractNode>
 
   <AbstractNode Name="BoundValuePlaceholderBase" Base="BoundExpression">
-    <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+    <Field Name="Type" Type="TypeSymbol" Override="true"/>
```

Comment:
Removing ```Null="disallow"``` feels suspicious, the whole point of placeholders is to have type. However, I am at the very beginning of the review process, I might change opinion as I make more progress.